src/components/application_manager/test/commands/mobile/delete_file_test.cc

```diff
@@ -197,6 +197,28 @@ TEST_F(DeleteFileRequestTest, Run_InvalidFile_UNSUCCESS) {
   command_->Run();
 }
 
+TEST_F(DeleteFileRequestTest, Run_ValidFileNameWithGenericError_SUCCESS) {
+  const std::string kFileName = "read_only.txt";
```

Comment:
@wolfylambova22, local constants should be named as usual variables. Please, check everywhere.